NextGenConnect/JSON to HL7v2.xml

```diff
@@ -277,19 +287,17 @@ tmp[&apos;RXA&apos;][&apos;RXA.6&apos;] = immunization[&apos;doseQuantity&apos;]
 
 tmp[&apos;RXA&apos;][&apos;RXA.7&apos;] = immunization[&apos;doseQuantity&apos;][&apos;code&apos;];
 
-//RXA-9 Administration Notes 00 -&gt; Administered, 01 -&gt; Record
+//RXA-9 Administration Notes 
 
 tmp[&apos;RXA&apos;][&apos;RXA.9&apos;] = 00;
 
 
-//RXA-10 Administering Provider -- Need a reference to actor
+//RXA-10 Administering Provider -- Reference to Practioner
```

Comment:
Similarly to Organization above, is this in scope for this PR?

Reply:
There is a card to add the extra resources to the bundle in the backlog.

Reply:
Which card is it? Could you include a link to it in your reply?

Reply:
https://mass-immunization-system.atlassian.net/jira/software/projects/MVS/boards/2?selectedIssue=MVS-530

Reply:
This is fixed.